domains/certificates/Query.API/API.AppTests/Infrastructure/RabbitMqContainer.cs

```diff
@@ -0,0 +1,32 @@
+using System.Threading.Tasks;
+using DotNet.Testcontainers.Builders;
+using DotNet.Testcontainers.Configurations;
+using DotNet.Testcontainers.Containers;
+using Xunit;
+
+namespace API.IntegrationTest.Infrastructure;
+
+public class RabbitMqContainer : IAsyncLifetime
+{
+    private readonly RabbitMqTestcontainer testContainer;
+
+    public RabbitMqContainer() =>
+        testContainer = new TestcontainersBuilder<RabbitMqTestcontainer>()
```

Comment:
Which docker image and version of RabbitMQ will this spin up? Is it possible to set it?
In production, the RabbitMQ version is determined by the RabbitMQ operator. For the version we are using, this will use 
this docker image+tag: "bitnami/rabbitmq:3.10.14-debian-11-r1"

Reply:
It will spin up "rabbitmq:3.7.28" - are you suggesting being specific on the image? 
Then we will need to maintain it

Reply:
Yes, I am suggesting that it set to the same image as will be used in production. The same also goes for the docker-compose image for RabbitMQ.
Unfortunately, it is determined by the version of RabbitMQ operator and is defined in eo-base-environment. So it will be a "magic string", but cannot see a way out of solving that right now.

Reply:
This will introduce some errors with M1 processors